playbook/controllers.go

```diff
@@ -143,6 +144,22 @@ func HandlePlaybookRun(c echo.Context) error {
 		return c.JSON(http.StatusInternalServerError, api.HTTPError{Error: err.Error(), Message: "failed to create playbook run"})
 	}
 
+	go func() {
+		// Create a new context because the original request context will be cancelled immediately
```

Comment:
Why does the context need to remain active after approving a run ? The run should be picked up in the background, and not synchronously during this call

Reply:
It doesn't remain active after approving a run?

It's also run asynchronously.

Reply:
It shouldn't be be kicked off by the http request - the http request updates the db and then the normal notify mechanism's should schedule it to run - the scheduling is pretty simple now, but we could have dedicated runners, pools of runners, runners with specific tags, etc..